ui/js/controllers/logviewer.js

```diff
@@ -376,6 +401,21 @@ logViewerApp.controller('LogviewerCtrl', [
             });
         }
 
+        function displayErrorLines(errors) {
+            var css = errors
+              .map(({ line_number }) => `a[id="${line_number + 1}"]+span`)
+              .join(',')
+              .concat('{background-color:red;color:white}');
```

Comment:
I also think that `background-color: pink; color: red;` has a more appealing look (for an error).